### PR TITLE
Pokemon Weedle was added to the pokemon list.

### PR DESCRIPTION
**Pull request by AndresMamaniMendoza**

### CODE REVIEW

**Review by mhiliri — APPROVED** [review]

**Comment by mhiliri** [comment]
Please fix the merge conflict

File: index.html [hunk 1]
```diff
@@ -32,6 +32,12 @@ <h1>Pokedex</h1>
       <td>Ivysaur</td>
       <td>Monica Hiliri</td>
     </tr>
+    <tr>
```
Comment (mhiliri) [inline index.html]: Please fix the merge conflict

**Review by mhiliri — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AndresMamaniMendoza) [reply index.html]: Conflicts were solved.

File: index.html [hunk 2]
```diff
@@ -54,6 +54,12 @@ <h1>Pokedex</h1>
          <td>Butterfree</td>
          <td>Jonathan Baldelomar</td>
          </tr>
+         <tr>
+            <td><center>13</center></td>
+            <td><img src="images/013Weedle.png"></td>
+            <td>Weedle</td>
+            <td>Andres Mamani</td>
+         </tr>
```
Comment (mhiliri) [inline index.html:68]: Please update you branch with main branch and fix the merge conflict

**Review by mhiliri — APPROVED** [review]

**Pull request merged**